engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -220,6 +220,17 @@ <h4 translate> Sync Settings </h4>
         </p>
       </div>
 
+      <div class="checkbox" ng-if="repository.content_type === 'yum'">
+        <label>
+          <input id="auto_enabled" name="auto_enabled" ng-model="repository.auto_enabled" type="checkbox"/>
+          <span translate>Enable on Registration</span>
+        </label>
+
+        <p class="help-block" translate>
+          Selecting this option will automatically enable this Repository when a Host registers and consumes this content pool.
```

Comment:
Is "content pool" the right term?  Would it be more user friendly to say "subscription?"